test/Sentry.Tests/Protocol/SentryEventTests.cs

```diff
@@ -29,111 +39,41 @@ public void SerializeObject_AllPropertiesSetToNonDefault_SerializesValidObject()
                 },
                 Modules = { { "module_key", "module_value" } },
                 Release = "release",
-                SentryExceptions = new[] { new SentryException { Value = "exception_value" } },
+                // SentryExceptions = new[] { new SentryException { Value = "exception_value" } },
```

Comment:
Setting this breaks the test with:

`System.Runtime.Serialization.SerializationException: Member 'Message' was not found.`

Even if simply setting an empty list! :gofigure: